src/scripts/console.art

```diff
@@ -179,8 +179,7 @@ lookup: function [what :string :literal :regex :type][
     unstack.discard size stack
 ]
 
-auto: #[complete: true, hints: true]
-Repl: #[verbose?: true]
+Repl: #[complete?: true, hints?: true, verbose?: true]
```

Comment:
`complete?` or `autocomplete?`

Reply:
@Eloitor You literally beat me by 2 seconds haha.

Well, before we had `auto\complete`, so it was like a wordplay that said "autocomplete".

I woudl change it to `autocomplete?` (`autocompletion?` would also be ok! 😉 )